File: src/main/java/dev/lbuddyboy/flash/user/model/PlayerInfo.java
package dev.lbuddyboy.flash.user.model;

import dev.lbuddyboy.flash.FlashLanguage;
import lombok.AllArgsConstructor;
import lombok.Data;
import org.bukkit.inventory.ItemStack;

import java.util.HashMap;
import java.util.List;
import java.util.Map;
import java.util.UUID;

@AllArgsConstructor
@Data
public class PlayerInfo {

    private boolean pmsOn, claimedNameMC, offlineInventoryEdited;
    private UUID reply;
    private long lastRequestSent, lastMessageSent;
    private List<UUID> readNotifications;

}

File: src/main/java/dev/lbuddyboy/flash/user/model/PlayerInfo.kt
package dev.lbuddyboy.flash.user.model

import lombok.AllArgsConstructor
import lombok.Data
import java.util.*

@AllArgsConstructor
@Data
class PlayerInfo {
    private val pmsOn = false
    private val claimedNameMC = false
    private val offlineInventoryEdited = false
    private val syncCode: Long = 0
    private val reply: UUID? = null
    private val lastRequestSent: Long = 0
    private val lastMessageSent: Long = 0
    private val readNotifications: List<UUID>? = null
}